tls/s2n_tls13_handshake.c

```diff
@@ -317,6 +317,19 @@ static int s2n_tls13_handle_master_secret(struct s2n_connection *conn)
     return S2N_SUCCESS;
 }
 
+static int s2n_tls13_handle_resumption_master_secret(struct s2n_connection *conn)
+{
+    s2n_tls13_connection_keys(keys, conn);
+    
+    struct s2n_hash_state hash_state = {0};
```

Comment:
Do we need to worry about leaking this?

Reply:
hash_state is stored on the connection and is eventually freed once the connection is freed (s2n_connection_free).